packages/runtime/runtime-utils/src/handles.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * JSON serialized form of an IFluidHandle
+ * @internal
+ */
+export interface ISerializedHandle {
+	// Marker to indicate to JSON.parse that the object is a Fluid handle
+	type: "__fluid_handle__";
+
+	// URL to the object. Relative URLs are relative to the handle context passed to the stringify.
```

Comment:
The URL is always an absolute path now.

Reply:
True

Reply:
Well... there are still old docs.  But I can update this if you want